contracts/src/libraries/LPMath.sol

```diff
@@ -756,9 +756,17 @@ library LPMath {
                 // proceeds couldn't be calculated.
                 return 0;
             }
-            uint256 presentValue = calculatePresentValue(
+
```

Comment:
Nit: Remove this newline. This falls into the broader theme of commenting code blocks. I see the value of separating this, so you can also remove the part of the above comment that pertains to the new block and comment this separately.